Content.Server/Atmos/Piping/Trinary/EntitySystems/GasMixerSystem.cs

```diff
@@ -236,5 +237,20 @@ private void OnMixerAnalyzed(EntityUid uid, GasMixerComponent component, GasAnal
 
             args.DeviceFlipped = inletOne != null && inletTwo != null && inletOne.CurrentPipeDirection.ToDirection() == inletTwo.CurrentPipeDirection.ToDirection().GetClockwise90Degrees();
         }
+
+
+        // Frontier - Start: Enable mixers at roundstart
```

Comment:
Remove extra blank line
```suggestion

        // Frontier - Start: Enable mixers at roundstart
```